index.html

```diff
@@ -16,23 +16,150 @@
-                <h1>Ram Kumar Karuppusamy</h1>
-                <p>Hello! I am a full-stack web developer.I can built responsive websites. Please look through my
+            <header>
+                <h2 class="section-header">Ram Kumar Karuppusamy</h2>
+                <p class="section-content">Hello! I am a full-stack web developer.I can build responsive websites.
+                    Please look through my
                     previous projects <br>If you like my work and want to hire me for projects, please contact me</p>
             </header>
-            <button class="button-contact">Start Collaboration</button>
+            <button class="button-generic">Start Collaboration</button>
             <!-- design art added in the background using css -->
         </section>
+        <!-- porjects information section -->
+        <section class="projects">
+            <h2 class="section-header">Projects</h2>
+            <div class="projects-container">
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 21.png" alt="open laptop top view" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 1</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
+                            <div class="tech-item">Ruby on Rails</div>
+                            <div class="tech-item">Javascript</div>
+                        </div>
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 22.png" alt="open laptop and a mobile" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 2</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
+                            <div class="tech-item">Ruby on Rails</div>
+                            <div class="tech-item">Javascript</div>
+                        </div>
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 26.png" alt="open laptop tilted to right" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 3</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
```

Comment:
[ ] The list of project technologies should be a list `ul` and each of them a `li`.kindly fix that 🤓